bin/src/worker.rs

```diff
@@ -113,7 +123,10 @@ pub fn begin_worker_process(
         max_command_buffer_size,
     );
 
-    worker_to_main_channel.blocking();
+    // DISCUSS: should we propagate the error instead of printing it?
+    if let Err(e) = worker_to_main_channel.blocking() {
```

Comment:
Do we know, why do we need to set the channel blocking? I do not get it :/

Reply:
I think it's in order to wait for the worker config and be sure to not miss it